components/Hero.js

```diff
@@ -13,7 +13,7 @@ const HeroContainer = styled.div`
   padding-bottom: 69%;
   background: url(/assets/hero_illustration.svg);
   background-repeat: no-repeat;
-  background-position: center;
+  background-position: top;
   background-size: contain;
   padding-top: 134px;
```

Comment:
Would recommend removing padding-top as well. This used added to shift down the hero, but now without the banner it's shifted down too much (e. g. nwPlus text is a bit low and might be way too low on smaller screens)

Reply:
Good catch, I didn't notice that